packages/front-end/components/PowerCalculation/PowerCalculationContent.tsx

```diff
@@ -75,9 +85,16 @@ const AnalysisSettings = ({
           <div className="col-7">
             <h2>Analysis Settings</h2>
             <p>
-              {params.nVariations} Variations · Frequentist (Sequential Testing{" "}
-              {params.statsEngine.sequentialTesting ? "enabled" : "disabled"}) ·{" "}
-              <Link href="#" onClick={() => setShowStatsEngineModal(true)}>
+              {params.nVariations} Variations ·{" "}
+              {engineType[params.statsEngineSettings.type]} (Sequential Testing{" "}
+              {params.statsEngineSettings.sequentialTesting
```

Comment:
@romain-growthbook we should only show `(Sequential Testing ...)` for Frequentist engine, regardless of whether it is enabled or disabled for now.

Reply:
@romain-growthbook , in the screenshot you shared, `Effect size` follows `Conversion rate` for the top metric, but precedes `Mean` for the bottom metric.   Can you please ensure `Effect size` always comes first?